lib/src/react16_suggestors/react_dom_render_migrator.dart

```diff
@@ -32,6 +32,12 @@ class ReactDomRenderMigrator extends GeneralizingAstVisitor
       return ancestor is CompilationUnit;
     });
 
+    ImportDirective overReactImport = importList.directives.lastWhere(
+        (dir) =>
+            dir is ImportDirective &&
+            dir.uri?.stringValue == 'package:over_react/over_react.dart',
```

Comment:
Can you also check for a `package:web_skin_dart/ui_core.dart` import since that exports `over_react.dart`?

More people use that than `over_react.dart` directly.

Reply:
fer sher fer sher